51 유저정보 네임 추가

chlalswns200 opened the pull request:
## :information_desk_person: 간단 소개
UserEntity에 name 컬럼 추가를 위한 작업을 했습니다
username -> 닉네임 역할
name -> 실명 역할

## :heavy_check_mark: 작업 내용 설명
entity에 name 추가하고
dto 클래스에도 name 컬럼 추가했습니다.

## :bulb: 참고사항
작업 하다 보니
닉네임 중복 체크 이슈#55 를 브랜치를 따로 안파고 여기서 작업해버렸습니다
중복체크 코드도 같이 봐주시면 감사하겠습니다ㅎㅎ

chlalswns200 committed:
refacto: UserEntity name 컬럼 추가

chlalswns200 committed:
test: join 기능 테스트 name도 받도록 수정

geuun reviewed (approved):
리뷰 완료하였습니다!

고생하셨습니다 :) !

한 분 더 리뷰 해주시면 병합 진행하겠습니다!

chlalswns200 committed:
feat: 닉네임 중복 체크 api 추가
Security
check 권한 열어주기
Controller
"/api/v1/users/check" 추가 성공하면 string 반환
service
email 조회 로직 address 들어가 있던거 수정
중복 체크를 하는 메서드 추가

chlalswns200 committed:
test: 닉네임 중복 체크 api 테스트
성공
실패 - 닉네임 중복

ahn-ji0 reviewed (approved):
확인했습니다~!

ahn-ji0 commented on src/main/java/com/dev/museummate/controller/UserController.java:
```diff
@@ -37,5 +37,11 @@ public Response<String> logout(@RequestBody UserReissueRequest userLogoutRequest
     }
 
+    @PostMapping("/check")
+    public Response<String> logout(@RequestBody UserCheckRequest userCheckRequest) {
```
메소드명 logout으로 되어있습니다!

ahn-ji0 replied:
그리고 혹시 PostMapping으로 하신 이유가 있으실까요?

chlalswns200 replied:
세상에 메서드명 수정하겠습니다!
중복 체크할 닉네임을 body에 받아서 그렇습니다
param으로 받아도 문제 없을거 같긴한데 일단 post로 했습니다

geuun commented:
병합 진행하겠습니다!!
